README.md

```diff
@@ -1,4 +1,7 @@
-# truex-tvos-integrations
-Documentation and reference apps for TrueX's tvOS based integration library.
+# true[X] tvOS Integrations
+
+Documentation and reference apps for true[X]'s tvOS-based integration library.
 
 The current version of the integration documentation can be found here: [TruexAdRenderer-tvOS Documentation](DOCS.md).
+
+The reference app can be found here: [truex-tvos-google-ad-manager-reference-app](https://github.com/socialvibe/truex-tvos-google-ad-manager-reference-app)
```

Comment:
Super-nit: may want to avoid language to imply this is the authoritative reference (given it depends on the IMA SDK). May just want to say "An example integration using the Google Ad Manager SDK for ad serving can be found here"

Reply:
True, sounds good.